packages/plugin/src/modules/options.js

```diff
@@ -97,14 +97,18 @@ export const options = {
         requestable: false,
         savable: true,
     },
-    hideEmptyColumns: { defaultValue: false, requestable: true, savable: true },
-    hideEmptyRows: { defaultValue: false, requestable: true, savable: true },
+    hideEmptyColumns: {
+        defaultValue: false,
+        requestable: false,
+        savable: true,
+    },
+    hideEmptyRows: { defaultValue: false, requestable: false, savable: true },
     skipRounding: { defaultValue: false, requestable: true, savable: true },
     numberType: { defaultValue: 'VALUE', requestable: false, savable: true },
     showHierarchy: { defaultValue: false, requestable: true, savable: true },
     legendSet: { defaultValue: undefined, requestable: false, savable: true },
     legendDisplayStrategy: {
-        defaultValue: undefined,
+        defaultValue: 'FIXED',
```

Comment:
What does `legendDisplayStrategy` and `legend` have to do with this fix?

Reply:
Nothing, but we should keep the 2 files aligned until we can use a shared one, otherwise it will get confusing to know which one is the "right" one.

Reply:
@martinkrulltott one is in /app, the other is in /plugin